swagger.json

```diff
@@ -182,9 +182,37 @@
         "parameters": [
           {
             "in": "body",
-            "name": "comment_id",
+            "name": "body",
```

Comment:
Did you add these lines to the swagger doc?

Reply:
@Qausim  yes, i did, it was my previous task, i was corrected by oluchi that the documentation was not properly done, it omitted that part so i had to add them